__tests__/command_helpers/checkCLIForUpdates.ts

```diff
@@ -1,6 +1,6 @@
 import checkCLIForUpdates from '../../src/extensions/functions/checkCLIForUpdates'
 
-import * as context from 'gluegun/toolbox'
+const context = require('mockContext')
```

Comment:
Is this using `require` syntax b/c it cannot be hoisted?

Reply:
looks good to me

Reply:
@FrederickEngelhardt good question. I just followed the existing pattern in the repo. I tried to keep my changes small and focused.